armi/nuclearDataIO/tests/test_xsLibraries.py

```diff
@@ -127,7 +127,7 @@ def test_mergeFailsWithNonIsotxsFiles(self):
             os.remove(dummyFileName)
 
         with TemporaryDirectoryChanger():
-            dummyFileName = "ISOtopics.txt"
+            dummyFileName = "ISO[]"
```

Comment:
What's the reason for this change?

Reply:
I wanted to allow files to have lower-case letters, e.g. `ISOAa` or `ISOBa`. This was previously not recognized as an ISOTXS file.